README.md

```diff
@@ -177,44 +177,64 @@ Example of a partial CR:
 
 ### Mounting and injecting binding information
 
-This specification allows for data to be mounted using volumes or injected using environment variables.  The best practice is to mount any sensitive information, such as passwords, since that will avoid accidentally exposure via environment dumps and subprocesses.  Also, binding binary data (e.g. .p12 certificate for Kafka) as an environment variable might cause a pod to fail to start (stuck on `CrashLoopBackOff`), so it advisable for backing services with such binding data to mark it with `bindAs: volume`.
+#### Mounting data
+Implementations of this specification **MUST** mount the binding data into the consuming application container in the following location:
+
+```
+<mountPathPrefix>/<service-id>/secret/<persisted_secret>
+```
+
+Where:
+* `<mountPathPrefix>` defaults to `/platform/bindings` if not specified in the `ServiceBinding` CR via the `mountPathPrefix` element.  The environment variable `SERVICE_BINDINGS_ROOT` **MUST** be set to the value of `<mountPathPrefix>`, so that applications can always find this directory, and it is immutable.  This means that if another `ServiceBinding` CR wants to project itself into the same container, it **MUST** reuse the current `SERVICE_BINDINGS_ROOT` value.  
+* `<service-id>` is the `id` field of the corresponding `service` entry in the `ServiceBinding` CR.  If the `id` field is not present, the `name` field is used instead.  The `<service-id>` path **MUST** be unique between the services bound to a particular application.
+* `<persisted_secret>` represents a set of files where the filename is a Secret key and the file contents is the corresponding value of that key.
+
+Example:  `/platform/bindings/mongo-db/secret/MONGODB_HOST`
```

Comment:
A better example here wouldn't include the "kind" in the service id even if this is how it would be practically used.  You should also be thinking up front about the possibility that bindings aren't obviously environment variables.  How about `/platform/bindings/my-nosql-db/secret/grid-fs-database`?  This is actually a real-life binding for Mongo, but illustrates how the names of files on the filesystem don't immediately identify things.

Reply:
Note, this suggestion to remove the Mongo-obvious names, while retaining Mono-semantic names applies throughout the document.

Reply:
good point about `mongo-db`.  I will switch to something that doesn't suggest this field is the `kind`.  

Yes, definitely we will have very `ugly` binding names.   This is where the `dataMappings` / `custom mapping` conversation comes in.  So we will have the ugly names, as well as the "environment var friendly" name.  😄